examples/dropwizard/pom.xml

```diff
@@ -0,0 +1,148 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project
+        xmlns="http://maven.apache.org/POM/4.0.0"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>example</groupId>
+    <artifactId>dropwizard</artifactId>
+    <version>1</version>
+    <packaging>jar</packaging>
+
+    <name>Dropwizard Jib Example</name>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+
+        <maven.compiler.source>1.8</maven.compiler.source>
+        <maven.compiler.target>1.8</maven.compiler.target>
+
+        <mainClass>example.JibExampleApplication</mainClass>
+
+        <dropwizard.version>1.3.7</dropwizard.version>
+        <dropwizard.server.config>src/main/jib/config/dropwizard.yml</dropwizard.server.config>
+        <dropwizard-template-config.version>1.5.0</dropwizard-template-config.version>
+
+        <jib.to.image>gcr.io/REPLACE-WITH-YOUR-GCP-PROJECT/image-built-with-jib</jib.to.image>
+        <jib.container.appRoot>/app</jib.container.appRoot>
+        <jib-maven-plugin.version>0.10.1</jib-maven-plugin.version>
+    </properties>
+
+    <profiles>
+        <profile>
+            <!-- Isolating Jib as a Maven Profile so that the config file
```

Comment:
Not really isolated in some sense this this is active by default. If it's active by default, I wonder what's really good to have a profile. Because of overriding `dropwizard.server.config`, `./mvnw exec:java` fails.

Reply:
I meant more like "separating". For example, if `mvn -Pjar` were to be added. 

Maybe that should be an excercize left up to the reader, though

Reply:
Ah, yeah, makes sense. Whatever you do, just make sure the commands in `README.md` work.

BTW, is the profile necessary? Configure the right paths to `dropwizard.yml` to `exec-maven-plugin` and `jib-maven-plugin`, and they should work together, right?

Reply:
>they should work together, right?

_Individually_ the commands in the README should work. Not _together_, unless you're saying something else.

Reply:
Yeah, the individual commands should work with jib-maven-plugin and exec-maven-plugin configs working together in the default profile.